libr/main/rasm2.c

```diff
@@ -191,6 +191,9 @@ static void rarch2_list(RAsmState *as, const char *arch) {
 	RListIter *iter, *iter2;
 	const char *feat;
 	PJ *pj = NULL;
+	if(!as){
```

Comment:
Probably better to use r_return_if_fail because that shouldnt happen

Reply:
There is only one call and it was checked before the call, no need to check again